docs/quickstart.rst

```diff
@@ -156,7 +195,7 @@ Fairlearn also allows us to quickly plot these metrics from the
     :start-after: # Analyze metrics using MetricFrame
     :end-before: # Customize plots with ylim
 
-.. figure:: auto_examples/images/sphx_glr_plot_quickstart_001.png
+.. figure:: auto_examples/images/sphx_glr_plot_quickstart_diabetes_001.png
```

Comment:
I don't see anything rendered. I think it should be just `...plot_quickstart_001.png`?

Reply:
After changing the path to _images/ as we discussed on the call today, the CircleCI artifacts show all the plots rendering.